experiments/amg2023/openmp/ramble.yaml

```diff
@@ -36,8 +37,18 @@ ramble:
             omp_num_threads: '{threads_per_node_core} * {n_nodes}'
           experiments:
             amg2023_omp_{compilers}_problem1_{n_nodes}_{omp_num_threads}_{px}_{py}_{pz}_{nx}_{ny}_{nz}:
+              modifiers:
+                - name: caliper
+                  mode: spot 
+              success_criteria:
```

Comment:
@rfhaque Do we have to have a success criteria, even if we use the output of this modifier only in post-processing?

Reply:
> @rfhaque Do we have to have a success criteria, even if we use the output of this modifier only in post-processing?

@pearce8 We still need some way to assess the success/failure of the experiment using "ramble workspace analyze". We use the modifier output in post-processing to extract the FOM and use that in the success criteria.